pages/box_buster.vue

```diff
@@ -135,6 +144,15 @@ export default {
   },
   created() {},
   methods: {
+    isBarcodeDuplicate(barcode) {
+      if (
+        this.barcodes_scanned.indexOf(barcode) !==
```

Comment:
What about using `.includes()` instead?

Reply:
includes will match with itself in the array returning true even if its only in there once, this way checks if itself is the last and only one in array

Reply:
Gotcha.